src/main/java/nl/_42/boot/csv/CsvController.java

```diff
@@ -76,7 +76,7 @@ private CsvResult upload(InputProvider provider, String type, CsvProperties prop
         try (InputStream is = provider.getInputStream()) {
             return csvService.load(is, type, properties);
         } catch (RuntimeException | IOException e) {
-            log.error("Could not load CSV file", e);
+            log.info("Could not load CSV", e);
```

Comment:
ipv waar je nu overal info van hebt gemaakt, warn doen? Of heb je dan nog steeds gezeur in je monitoring?